### PR TITLE
#162949010 Users should be able to track comment edit history

### PR DESCRIPTION
**Pull request by Nyakaru**
#### What does this PR do?
This PR merges the track comment edit history feature branch to develop

#### Description of Task to be completed?
Allow authenticated users to track comment edit history

#### How should this be manually tested?
Run the server:
`python manage.py runserver`

From postman:
   
Get the edit history of a comment by sending a get request to 
 `/api/v1/<slug>/comments/<int:id>/history/` end point

#### What are the relevant pivotal tracker stories?
[162949010](https://www.pivotaltracker.com/story/show/162949010)

##### Get comment edit history of a comment that has updates
screenshot 2019-01-30 at 11 59 46

##### Get comment edit history of a comment that has no updates
screenshot 2019-01-30 at 20 37 17

##### Get comment edit history of a non existing comment
screenshot 2019-01-30 at 20 37 34

##### Get comment edit history of a non existing article
screenshot 2019-01-30 at 12 03 07

##### Unauthenticated user getting comment edit history
screenshot 2019-01-30 at 13 45 44

### CODE REVIEW

**Review by Hesbon5600 — COMMENTED** [review]
Great implementation of DRY code.

**Review by eduhmik — COMMENTED** [review]
Hey @Nyakaru, nice way to implement how to track edit history of comments using django simple history. The work look clean and works fine when tested.

**Review by JoshuaKodhe — COMMENTED** [review]
I like how you researched on the task and used an inbuilt library. 
This has simplified the code. 

The code is also readable and easy to go through.

**Review by charitymarani — APPROVED** [review]

**Review by Nicanor008 — APPROVED** [review]

File: authors/apps/articles/tests/base_test.py [hunk 1]
```diff
@@ -123,6 +123,18 @@ def add_comment(self, slug):
...
+        response = self.add_comment(slug)
+        id = response.data['id']
+        response = self.client.put(
+            (self.specific_comment(slug, id)),
+            data={'comment': {'body': 'New comment'}},
+            format='json',
+            HTTP_AUTHORIZATION="Token " + token)
+        return response
+
     def get_comment(self, id):
```
Comment (kzyangiro) [inline authors/apps/articles/tests/base_test.py]: @Nyakaru Update_comment should come after get_comment

**Review by kzyangiro — CHANGES_REQUESTED** [review]
@Nyakaru **comment not found** and **no edit history for this comment** are in plain text, you could kindly maintain using json format for the responses to have a uniform order

**Comment by Nyakaru** [comment]
> @Nyakaru **comment not found** and **no edit history for this comment** are in plain text, you could kindly maintain using json format for the responses to have a uniform order

Worked on it. Kindly you could check the updated screenshots.

**Review by kzyangiro — APPROVED** [review]

**Pull request merged**